### ISSUE TITLE
wp-to-diaspora dont't post to diaspora anymore

### ISSUE THREAD

**Issue by elektrollart** [gutobenn/wp-to-diaspora#125]
Since this morning, the plugin doesn't post the Word Press posts to diaspora anymore, I didn't change anything, all as always. I removed the plugin and reinstalled it. No changes. I don't get any errors, I just see that the mark for "post to diaspora" isn't set, after I release my post on Word Press. 

* Word Press Version = 4.5.3
* wp-to-diaspora Version = 1.7.0 (I tested with the source from GitHub too)
* Pod: nerdpol.ch

**Comment by noplanman** [comment]
I have no issues on nerdpol.ch, will inspect further when I get home.

Any extra details you could give us?

**Comment by noplanman** [comment]
What other plugins do you have installed? Might be a clash with one of them.

**Comment by elektrollart** [comment]
These are the installed plugins.

* Disable Comments
* Open Graph
* Post Thumbnail Editor
* Share on Diaspora
* Ultimate Category Excluder
* WP to diaspora
* WP to Twitter
* WP-PageNavi
* WP-Spreadplugin

**Comment by elektrollart** [comment]
I kicked WP to Twitter, now it works. There a problems with this plugin and wp to diaspora.

**Comment by noplanman** [comment]
After digging into WordPress Core, it seems to be a WordPress bug.

Have filed an issue [here](https://core.trac.wordpress.org/ticket/37679), and seen that it's been an issue for a [very long time](https://core.trac.wordpress.org/ticket/17817).

Sure hope this gets fixed soon.

If you need both plugins to work together for now, you can simply adjust the priority of the WP to diaspora* actions [here](https://github.com/gutobenn/wp-to-diaspora/blob/v1.7/lib/class-post.php#L127).

You can set it to 11 like so:
```php
add_action( 'save_post', array( $instance, 'post' ), 11, 2 );
```

This just has to be lower than 15 (that WP To Twitter uses). You'd then have to make sure that the Twitter post is correctly created.

**Comment by noplanman** [comment]
@elektrollart Good news!

On the one hand I forgot to mention that I updated the plugin to 1.7.1, which now works together with WP to Twitter.

On the other hand, this entire problem in WP core seems to be fixed and will hopefully be part of WP 4.7 🎉 ([check here](https://core.trac.wordpress.org/ticket/17817#comment:214))

**Comment by elektrollart** [comment]
<3
